packages/babel-types/src/definitions/experimental.js

```diff
@@ -226,6 +243,14 @@ defineType("Import", {
 
 defineType("ImportAttribute", {
   visitor: ["key", "value"],
+  fields: {
+    key: {
+      validate: assertNodeType("Identifier"),
+    },
+    value: {
+      validate: assertNodeType("StringLiteral"),
+    },
+  },
```

Comment:
This is _technically_ a breaking change, but this node is so new that I'd be fine with considering it a bug fix.

Reply:
I think generally such changes should be considered a bug fix because the validator should try its best to align with the parser behaviour.